README.rst

```diff
@@ -39,19 +40,59 @@ dhtnode and the former starts all instances configured on the server. Configured
 instances are auto-detected by checking for the presence of ``config.ini`` files
 in ``/srv/dhtnode/dhtnode-*/etc/``.
 
+
+Systemd
+-------
+
+Two systemd unit templates exist: ``/lib/systemd/system/dht@.service`` and
+``/lib/systemd/system/dhtdum@.service``. These files are service templates and
+they can be used to start any number of enabled instances on the machine. Enabling
+instances is done via ``sudo systemctl enable dht@N`` where ``N`` is the instance
+number. To enable first 8 instances, one can use its shell and do
+``sudo systemctl enable dht@{1..8}``.
```

Comment:
Enabling instances will make them start automatically on every reboot?

Reply:
Yes. `enable` will hook up the service into the given (multi-user) target so it will start it automatically as soon as multiuser target is up and the dependencies are up. It will not start the services, for that you need `systemctl start dht@{1..8}`